pkg/validation/validation-rule.md

```diff
@@ -0,0 +1,58 @@
+### Id and Name:
+`^[a-z0-9]([-a-z0-9]*[a-z0-9])?$`
+
+The restriction is added to allow easy translation to K8s resource names, and also to have consistent rules for both name and id fields
```

Comment:
We need a punctual mark here or new line, because according to the markdown rules, it's going to be inlined with the next line
![Screenshot_20201210_093047](https://user-images.githubusercontent.com/5887312/101735406-88ed7400-3aca-11eb-8c86-56a64e422688.png)
```suggestion
The restriction is added to allow easy translation to K8s resource names, and also to have consistent rules for both name and id fields

```